acceptance/setup/git/pre-suite/030_PuppetMasterSanity.rb

```diff
@@ -1,6 +1,6 @@
 test_name "Puppet Master sanity checks: PID file and SSL dir creation"
 
-pidfile = '/var/lib/puppet/run/master.pid'
+pidfile = '/var/run/puppetlabs/agent/master.pid'
```

Comment:
So we'll split the `pidfile` in defaults.rb to `agent_pidfile` and `master_pidfile`? I also just realized that we'll need to update every init script in ext as well.